services/validation/ValidationExtrasValidators.cfc

```diff
@@ -4,11 +4,18 @@
 
  component {
 
- 	public any function init() {
+	/**
+	 * @systemConfigurationService.inject systemConfigurationService
```

Comment:
Rather than do this, it would make sense to use `@presideService` on the component meta definition. It should also be a singleton.

Then you can just use `$getPresideSetting()` and any other super class methods and not have to inject the systemConfigurationService directly. So:

```cfc
/**
 * @singleton
 * @presideService
 * @validationProvider
 */
component {
// ...
}
```